Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.3.3'
+ruby '~>2.5.5'
```

Comment:
```suggestion
ruby '~> 2.5.5'
```

Reply:
Sorry, I did not see it was a suggestion, I pushed the modified commit.